src/renderer/components/depositActions/forms/Bond.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback } from 'react'
+
+import {
+  AssetAmount,
+  assetAmount,
+  AssetRuneNative,
+  assetToBase,
+  BaseAmount,
+  formatAssetAmount,
+  formatAssetAmountCurrency
+} from '@xchainjs/xchain-util'
+import { Form } from 'antd'
+import BigNumber from 'bignumber.js'
+import { useIntl } from 'react-intl'
+
+import { Input, InputBigNumber } from '../../uielements/input'
+import * as Styled from './Forms.styles'
+
+const getBondMemo = (thorAddress: string) => `BOND:${thorAddress}`
```

Comment:
^ Let provide such memo helper by `asgardex-util` (with another PR if you want) - new issue :eyes:  https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/issues/17